### ISSUE TITLE
removeLastPoint does not remove last point in draw interaction.

### ISSUE THREAD

**Issue by dbauszus-glx** [openlayers/openlayers#9985]
Using the removeLastPoint method in a draw interaction seems to remove the second last vertice as defined in the [code for the draw interaction](https://github.com/openlayers/openlayers/blob/4161831e299fbb9b16f6e37a7c8989a4cdaebb74/src/ol/interaction/Draw.js).

```
coordinates.splice(-2, 1); // I believe this should be .splice(-1,1);
```

**Comment by mike-000** [comment]
The final coordinate is the pointer position, the previous coordinates are the drawn vertices.  You can see it in action here https://codesandbox.io/s/draw-features-luyz6  If you click the open in new window icon in codesandbox so it can detect keyboard events hitting Esc will remove vertices from an in progress draw.

**Comment by dbauszus-glx** [comment]
Thanks for the information.

I tried now the keyboard events on different machines but it doesn't seem to recognize the ESC key input.

I was however able to workout how to remove the last vertice. The problem was a missing preventDefault event on the click in the context menu which should remove the last vertice. This created first a vertice then immediately removing the same vertice.

I noticed that the drawing is broken in the version 6 beta. Where should is post issues with v6?
